### PR TITLE
add notice file

### PR DESCRIPTION
**Pull request by MajdT51**
it must include a notice file. the notice template can be found under: 
https://github.com/catenax-ng/foss-example/blob/main/catenax-ng-repositories/NOTICE_template.md

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Notices
+
+This content is part of [CatenaX](https://catena-x.net). 
+
+* Project home: https://github.com/catenax-ng
+
```
Comment (jpbu) [inline NOTICE.md:6]: What about the "See the AUTHORS file(s) distributed with this work for additional information regarding authorship." in the template? We should check whether we need that as well, I don't see any template though.

File: NOTICE.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+https://www.apache.org/licenses/LICENSE-2.0.
+
+SPDX-License-Identifier: Apache-2.0
+
+## Source Code
+
+The project maintains the following source code repositories 
+in the GitHub organization https://github.com/catenax-ng:
+
+* https://github.com/catenax-ng/product-core-managed-identity-wallets
```
Comment (jpbu) [inline NOTICE.md:26]: Do we need a notice file in all of the listed repositories?

**Review by jpbu — COMMENTED** [review]
Overall looks good to me, two questions that we should clarify.

File: NOTICE.md (hunk 1 above)
Comment (MajdT51) [reply NOTICE.md:6]: Authors is optional, and Matthias didn't add it.

File: NOTICE.md (hunk 2 above)
Comment (MajdT51) [reply NOTICE.md:26]: @matthiashub-porsche what do you think? if yes then we probably need also to add security and dependency as well

Comment (matthiashub-porsche) [reply NOTICE.md:26]: That is right, I don't know yet what projects will be contributed to Eclipse besides this main repository, thus I would remove the two others for now

**Pull request merged**